.travis.yml

```diff
@@ -1,3 +1,4 @@
+# .travis.yml
```

Comment:
Is this line needed?

Reply:
Not sure what value shebangs brings here, so will remove it.

Reply:
Addressed.